.github/workflows/flowzone.yml

```diff
@@ -12,3 +12,5 @@ jobs:
     name: Flowzone
     uses: product-os/flowzone/.github/workflows/flowzone.yml@master
     secrets: inherit
+    with:
+      tests_run_on: '["windows-2019","ubuntu-20.04","macos-11"]'
```

Comment:
```suggestion
      runs_on: '["windows-2019","ubuntu-20.04","macos-11"]'
```

Reply:
@thgreasi this does not seems to work, am I doing something wrong ?